src/slider/README.md

```diff
@@ -1 +1,83 @@
-# Slider widget
+# @dojo/widgets/slider/Slider widget
+
+Dojo 2's `Slider` widget creates a range slider control with a styleable track, fill, and thumb.
+
+
+## Features
+
+- Horizontal or vertical slider with a track, fill, and single thumb
+- Output node with customizable text
+
+### Accessibility Features
+
+`Slider` uses the native `<input type="range">`as its base, which ensures built-in keyboard and screen reader accessibility. All common form field attributes (`disabled`, `invalid`, `readOnly`, `required`) may be set, as well as a visible or hidden label. The output node is associated with the input.
```

Comment:
"The output node is associated with the input." I'm not sure most readers will know what is meant by this.